cli/src/pcluster/schemas/validators.py

```diff
@@ -0,0 +1,184 @@
+    if fsx_imported_file_chunk_size and not fsx_import_path:
+        errors.append("When specifying 'imported_file_chunk_size', the 'import_path' option must be specified")
+
+    if fsx_export_path and not fsx_import_path:
+        errors.append("When specifying 'export_path', the 'import_path' option must be specified")
+
+    if fsx_auto_import_policy and not fsx_import_path:
+        errors.append("When specifying 'auto_import_policy', the 'import_path' option must be specified")
+
+
+def validate_persistent_options(errors, fsx_deployment_type, fsx_kms_key_id, fsx_per_unit_storage_throughput):
+    if fsx_deployment_type == "PERSISTENT_1":
+        if not fsx_per_unit_storage_throughput:
+            errors.append("'per_unit_storage_throughput' must be specified when 'deployment_type = PERSISTENT_1'")
+    else:
+        if fsx_kms_key_id:
+            errors.append("'fsx_kms_key_id' can only be used when 'deployment_type = PERSISTENT_1'")
+        if fsx_per_unit_storage_throughput:
+            errors.append("'per_unit_storage_throughput' can only be used when 'deployment_type = PERSISTENT_1'")
+
+
+def validate_backup_options(
+    errors,
+    fsx_automatic_backup_retention_days,
+    fsx_daily_automatic_backup_start_time,
+    fsx_copy_tags_to_backups,
+    fsx_deployment_type,
+    fsx_imported_file_chunk_size,
+    fsx_import_path,
+    fsx_export_path,
+    fsx_auto_import_policy,
+):
+    if not fsx_automatic_backup_retention_days and fsx_daily_automatic_backup_start_time:
+        errors.append(
+            "When specifying 'daily_automatic_backup_start_time', "
+            "the 'automatic_backup_retention_days' option must be specified"
+        )
+    if not fsx_automatic_backup_retention_days and fsx_copy_tags_to_backups is not None:
+        errors.append(
+            "When specifying 'copy_tags_to_backups', the 'automatic_backup_retention_days' option must be specified"
+        )
+    if fsx_deployment_type != "PERSISTENT_1" and fsx_automatic_backup_retention_days:
+        errors.append("FSx automatic backup features can be used only with 'PERSISTENT_1' file systems")
+    if (
+        fsx_imported_file_chunk_size or fsx_import_path or fsx_export_path or fsx_auto_import_policy
+    ) and fsx_automatic_backup_retention_days:
+        errors.append("Backups cannot be created on S3-linked file systems")
+
+
+def validate_storage_type_options(
+    errors, fsx_storage_type, fsx_deployment_type, fsx_per_unit_storage_throughput, fsx_drive_cache_type
+):
+    if fsx_storage_type == "HDD":
+        if fsx_deployment_type != "PERSISTENT_1":
+            errors.append("For HDD filesystems, 'deployment_type' must be 'PERSISTENT_1'")
+        if fsx_per_unit_storage_throughput not in FSX_HDD_THROUGHPUT:
+            errors.append(
+                "For HDD filesystems, 'per_unit_storage_throughput' can only have the following values: {0}".format(
```

Comment:
I mean we cannot talk about `per_unit_storage_throughput` because now the parameter is `PerUnitStorageThroughput`. To avoid this misalignements we should show an error message like:

`For HDD filesystems, the per unit storage throughput can only have the following values...` or something like this.